src/flamegpu/sim/Simulation.cu

```diff
@@ -118,9 +133,12 @@ void Simulation::printHelp(const char* executable) {
     printf("Usage: %s [-s steps] [-d device_id] [-r random_seed]\n", executable);
     printf("Optional Arguments:\n");
     const char *line_fmt = "%-18s %s\n";
-    printf(line_fmt, "-i, --in", "Initial state file (XML)");
-    printf(line_fmt, "-s, --steps", "Number of simulation iterations");
-    printf(line_fmt, "-r, --random", "RandomManager seed");
+    printf(line_fmt, "-i, --in <file.xml>", "Initial state file (XML)");
+    printf(line_fmt, "-s, --steps <steps>", "Number of simulation iterations");
+    printf(line_fmt, "-r, --random <seed>", "RandomManager seed");
+    printf(line_fmt, "-v, --verbose", "Verbose FLAME GPU output");
+    printf(line_fmt, "-t, --timing", "Output timing information to stdout");
+    printf(line_fmt, "    --no-timing", "Do not output timing information to stdout");
```

Comment:
Why have an option of on and a separate option for off? Surely one is redundant.

Reply:
I was probably thinking about making it opt-out but got distracted (to match current flame).